### ISSUE TITLE
[OPS] First "relation" option doesn't appear at preprint landing page after preprint is posted

### ISSUE THREAD

**Issue by alexxxmendonca** [pkp/pkp-lib#7117]
**Describe the bug**
When the first "relation" option is selected ("Preprint has not been submitted for publications"), it is not appearing at the preprint landing page after the preprint is posted, as opposed to the other 2 options.

**To Reproduce**
1. Select relation option "Preprint has not been submitted for publications":
![image](https://user-images.githubusercontent.com/4059679/121905331-6dc12280-cd00-11eb-9ebd-10d7f952ba82.png)
2. Post the preprint
3. Check the posted preprint landing page
4. Notice how the relation does not appear:
![image](https://user-images.githubusercontent.com/4059679/121905632-bd075300-cd00-11eb-859a-450779a7fee3.png)

**What application are you using?**
Open Preprint Systems 3.3.0.6

**Comment by NateWr** [comment]
I think that this is the intended behaviour. The notice is only expected to appear when the preprint has been submitted or published elsewhere. Otherwise, it has not.

If we want to show the relation status on every page, we can consider doing that. But we wouldn't want to show a big notice on every preprint. We'd need to reduce the prominence of this element on the page.

**Comment by alexxxmendonca** [comment]
If that's the intended behavior, it goes against recommendations for preprint servers such as ASAPBio's "Preprints in the public eye":

![image](https://user-images.githubusercontent.com/4059679/122050150-935e3280-cdb9-11eb-8977-006d6288ace7.png)

Source: https://asapbio.org/wp-content/uploads/2021/03/Preprints-in-the-Public-Eye-Preprint-Servers-infographic.pdf
More info: https://asapbio.org/public

**Comment by NateWr** [comment]
As I understand criteria 8, we are following the recommendation. We indicate "when a preprint has been published in a peer reviewed journal". It doesn't say to indicate when it has not been published.

That would accord with what I would expect from a UX perspective. Not published in a peer reviewed journal is the default, expected state of a preprint. And so a notice would only need to appear when something doesn't match this expectation.

**Comment by alexxxmendonca** [comment]
There is nothing in the interface that treats number 4 (the peer review status of the preprint).

Ideally servers should meet all 8 recommendations.

**Comment by alexxxmendonca** [comment]
OPS is doing numbers 1, 3, 4 (partially) and 8.

**Comment by NateWr** [comment]
Ah ok, number 4 won't be impacted in any way by the relations setting, because it identifies the peer review status of the preprint, not the item published in a peer reviewed journal.

I've updated the title of this issue to reflect the feature request as I understand it: the need for a preprint server to provide a statement, on the preprint landing page, covering some of these criteria. Does that look right?

**Comment by alexxxmendonca** [comment]
Yes, that is even better.

Still, on the matter of explicitly showing the first relation option, it is a matter of transparency but also: 

When authors submit a manuscript, there is nowhere where they can inform about the status of the preprint. We, as server managers, can't tell whether the manuscript has been submitted or not to a journal. We can only guess that it hasn't been submitted to a journal at the point of submission in our server, but that is nothing but a guess. We need authors to inform that to us.

When the preprint is posted, if the status "Preprint has not been submitted for publication" is shown, authors can contact us and correct that information. However, if the status is never shown, a preprint can remain with a wrong status indefinitely, because the authors may never know (given that the information is not public).

In other words, it's a way to keep the process as transparent as possible and minimize assumptions.

**Comment by NateWr** [comment]
@alexxxmendonca it sounds like we need to ask for this information during submission. Can you file a separate issue to address that?

**Comment by alexxxmendonca** [comment]
@NateWr done! #7134
